LocalPayment/src/main/java/com/braintreepayments/api/localpayment/LocalPaymentRequest.kt

```diff
@@ -0,0 +1,171 @@
+package com.braintreepayments.api.localpayment
+
+import com.braintreepayments.api.core.PostalAddress
+import org.json.JSONException
+import org.json.JSONObject
+
+/**
+ * Builder used to construct an local payment request.
+ */
+class LocalPaymentRequest
+/**
+ * @param hasUserLocationConsent is an optional parameter that informs the SDK
```

Comment:
Can we combine this comment block with the main class kdoc?

Reply:
+1 kdoc property annotations are great for readability.